hoodie-client/src/main/java/com/uber/hoodie/HoodieWriteClient.java

```diff
@@ -98,7 +99,7 @@
     private final HoodieWriteConfig config;
     private transient final HoodieMetrics metrics;
     private transient final HoodieIndex<T> index;
-    private transient final HoodieCommitArchiveLog archiveLog;
+    private transient final HoodieCommitArchiver archiver;
```

Comment:
Can you keep the name as HoodieCommitArchiveLog? That name is good enough and this clouds the changes done to that. In general - when making a big change like this - where correctness is more important to review. Keep the other changes to min/none. This will help the reviewer to focus on the right thing.